scripts/core/exp-command-buffer.yml

```
@@ -217,6 +217,51 @@ returns:
     - $X_RESULT_ERROR_OUT_OF_RESOURCES
 --- #--------------------------------------------------------------------------
 type: function
+desc: "Append a USM fill command to a command-buffer object"
+class: $xCommandBuffer
+name: AppendFillUSMExp
```

Also realised this is internally consistent but not externally consistent with `urEnqueueUSMFill`.

```suggestion
name: AppendUSMFillExp
```